src/QGCApplication.cc

```diff
@@ -558,7 +558,10 @@ bool QGCApplication::_initForNormalAppBoot()
     // Exit main application when last window is closed
     connect(this, &QGCApplication::lastWindowClosed, this, QGCApplication::quit);
 
-    _qmlAppEngine = toolbox()->corePlugin()->createRootWindow(this);
+
+    _qmlAppEngine = new QQmlApplicationEngine(this);
+    _qmlAppEngine->rootContext()->setContextProperty("multiVehicleManager", toolbox()->multiVehicleManager());
```

Comment:
Why are you pushing multiVehicleManager into the Qml context. It is already available from QGroundControl.multiVehicleManager.

Reply:
Removed, was not aware that this is exposed.